cloud-tenant-base-dependencies-enforcer/pom.xml

```diff
@@ -250,6 +250,7 @@
                                         <include>org.apache.httpcomponents:httpcore:4.4.13:jar:test</include>
                                         <include>org.apache.opennlp:opennlp-tools:1.9.3:jar:test</include>
                                         <include>org.apiguardian:apiguardian-api:1.1.0:jar:test</include>
+                                        <include>org.bouncycastle:bcutil-jdk15on:[${bouncycastle.version}]:jar:test</include>
```

Comment:
What makes it necessary to add it here?

Reply:
Application based tests may execute code that uses BouncyCastle